app/views/year_select/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<% content_for :page_title %>
+
+<section class="slab question-layout <%= controller_name.gsub("_", "-") %>-outer">
+  <div class="grid">
+    <div class="grid__item question-wrapper">
+      <h1 class="h2 spacing-below-5">
+        <%= t("views.year_select.show.title") %>
+      </h1>
+      <p class="spacing-below-15 spacing-above-0">
+        <%= t("views.year_select.show.subtitle") %>
+      </p>
+
+      <%= form_with model: @form, url: year_select_path, local: true, method: :patch, builder: VitaMinFormBuilder do |f| %>
```

Comment:
I realize that `VitaMinFormBuilder` is just copied from `vita-min` -- maybe we should call it just generic `CustomFormBuilder` (and rename the methods that refer to vita-min) or something like that. I realize this may cause issues with copy/pasting from vitamin down the road, so happy to have it as chore/clean up task after bulk of building is done, but please comment here with that ticket when you get a chance!

Reply:
I think changing these names will cause pain during development. I think it is reasonable to do when we are done, especially because this project has a definitive "done"

Reply:
Cool, pls create a chore to rename this in the future and put it in backlog!

Reply:
I feel like that is a lot of effort for something that is not providing a ton of value. Especially since there will not be continued work on this app and people working on this will likely have the historical context of what "vita-min" is.

Reply:
chatted & decided we can cross the renaming bridge if we repurpose pya to support more apps in the future